src/eventDispatchers/imageRenderedEventDispatcher.js

```diff
@@ -16,14 +17,17 @@ const onImageRendered = function(evt) {
         tool.mode === 'enabled')
   );
 
-  const brushTools = toolsToRender.filter(
-    tool => tool instanceof BaseBrushTool
+  const brushAndSegmentationTools = toolsToRender.filter(
+    tool =>
+      tool instanceof BaseBrushTool || tool instanceof BaseSegmentationTool
   );
 
-  if (brushTools.length > 0) {
+  if (brushAndSegmentationTools.length) {
     onImageRenderedBrushEventHandler(evt);
   }
 
+  onImageRenderedBrushEventHandler(evt);
```

Comment:
This has us calling the handler twice if we have any `brushAndSegmentationTools` or once if we have none?

Reply:
That's litterally a mistake, 2x speedup, nice.